linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -1085,7 +1106,10 @@ where
     let block_proposal = make_first_block(ChainId::root(1))
         .with_simple_transfer(Recipient::root(2), Amount::from_tokens(1000))
         .into_simple_proposal(&sender_key_pair);
-    assert!(worker.handle_block_proposal(block_proposal).await.is_err());
+    assert!(matches!(
+        worker.handle_block_proposal(block_proposal).await,
+        Err(WorkerError::ChainError(error)) if matches!(*error, ChainError::ExecutionError(ExecutionError::SystemError(SystemExecutionError::InsufficientFunding { .. }), ChainExecutionContext::Operation(_)))
```

Comment:
Please manually format this; `rustfmt` sometimes gives up. :sweat_smile:

Reply:
Sure! These do need beautification